ports/espressif/boards/waveshare_esp32_s3_pico/pins.c

```diff
@@ -25,6 +25,8 @@ STATIC const mp_rom_map_elem_t board_module_globals_table[] = {
     { MP_ROM_QSTR(MP_QSTR_IO19), MP_ROM_PTR(&pin_GPIO19) },
     { MP_ROM_QSTR(MP_QSTR_IO20), MP_ROM_PTR(&pin_GPIO20) },
     { MP_ROM_QSTR(MP_QSTR_IO21), MP_ROM_PTR(&pin_GPIO21) },
+    { MP_ROM_QSTR(MP_QSTR_IO33), MP_ROM_PTR(&pin_GPIO33) },
+    { MP_ROM_QSTR(MP_QSTR_IO34), MP_ROM_PTR(&pin_GPIO34) },
```

Comment:
These are not broken out on this particular board, or am I missing something? They are labeled in the schematic, but the module pins are not connected to anything.

Reply:
Sorry, never mind. I was confused with the GPIO43 and GPIO44 pins.